docs/whatsnew/2.1.rst

```diff
@@ -0,0 +1,95 @@
+    <SkyCoord( HeliographicStonyhurst: obstime=2021-01-01T00:00:00.000): (lon, lat, radius) in (deg, deg, AU)
+        (-34.46752135, 1.77496469, 1.50936573)
+     (d_lon, d_lat, d_radius) in (arcsec / s, arcsec / s, km / s)
+        (-0.00048971, 0.00060976, 19.54950062)>
+    >>> mars.velocity.norm()
+    <Quantity 19.56823076 km / s>
+
+However, `~sunpy.coordinates.frames.HeliographicStonyhurst` is a non-inertial frame that rotates over time.
+By transforming this coordinate to `~sunpy.coordinates.frames.HeliocentricInertial`, we can see that Mars's actual velocity is larger::
+
+    >>> mars.heliocentricinertial
+    <SkyCoord (HeliocentricInertial: obstime=2021-01-01T00:00:00.000): (lon, lat, distance) in (deg, deg, AU)
+        (-9.91128592, 1.77496469, 1.50936573)
+     (d_lon, d_lat, d_distance) in (arcsec / s, arcsec / s, km / s)
+        (0.04174239, 0.00060976, 19.54950058)>
+    >>> mars.heliocentricinertial.velocity.norm()
+    <Quantity 49.68592218 km / s>
+
+See :ref:`sunpy-coordinates-velocities` for more information.
+
+Alternatives for reprojecting a Helioprojective map
+===================================================
+The typical observation in `~sunpy.coordinates.frames.Helioprojective` coordinates does not contain full 3D information for the sources of emission, so an assumption needs to be made when transforming such coordinates to other coordinate frames.
+By default, SunPy assumes that the emission is coming from the surface of the Sun, which enables reprojections such as in the example :ref:`sphx_glr_generated_gallery_map_transformations_reprojection_different_observers.py`.
+However, this assumption is not appropriate for some observations, e.g., from coronagraphs.
+
+There is now a context manager (:meth:`~sunpy.coordinates.frames.Helioprojective.assume_spherical_screen`) to override the default assumption such that any 2D coordinates are interpreted as being on the inside of a large spherical screen.
+See the following example for how this context manager enables alternative reprojections.
+
+.. minigallery:: sunpy.coordinates.Helioprojective.assume_spherical_screen
+
+Performance improvements
+========================
+Several functions in `sunpy.map.maputils` have been significantly sped up with improved algorithms.
+
+In addition, `sunpy.map.Map.wcs` is now cached when the map metadata remains unchanged, significantly improving performance in applications which make mutiple requests for the map WCS (e.g. plotting), and reducing the number of repeated warnings thrown when metadata is missing.
+
+Support for GOES-16 and GOES-17 X-Ray Sensor (XRS) data
+=======================================================
+We have now included support for the GOES-16 and GOES-17 XRS 1-second data. This data can now be queried and downloaded with `sunpy.net.Fido` and the files read in and analysed as a `sunpy.timeseries.TimeSeries`.
+
+See the following example for how this can be achieved:
+
+(add example "goes_xrs_example")
```

Comment:
How can you link an example? (https://docs.sunpy.org/en/latest/generated/gallery/acquiring_data/goes_xrs_example.html#sphx-glr-generated-gallery-acquiring-data-goes-xrs-example-py)

Reply:
https://sphinx-gallery.github.io/stable/advanced.html#know-your-gallery-files

Reply:
thanks! hmm still a bit confused though (sorry my sphynx knowledge is not great!) - is the best way just to do a 
```
.. literalinclude:: ../examples/acquiring_data/goes_xrs_example.py
   :language: python
```

or should I just hyperlink it?

Reply:
`literalinclude` will literally include (😆) the code or a code snippet, but I'm guessing you don't want to list all the code from the example here? If you just want a link you can use \`:ref:\`sphx_glr_gallery_acquiring_data_goes_xrs_example.py\` (which I think is the correctly parsed link).

Reply:
Yep, I've just checked an old locally generated version. If you build the docs and go to `docs/generated/gallery/acquiring_data/goes_xrs_example.rst`, the reference is given on line 9 as `sphx_glr_generated_gallery_acquiring_data_goes_xrs_example.py`

Reply:
In case you need an example of how to type the reference, look at this line earlier in this file:
https://github.com/sunpy/sunpy/blame/842070404e970f6811463613feb2a73fc485961c/docs/whatsnew/2.1.rst#L65

Reply:
I just edited @dstansby 's earlier comment to get around the fiddling with backticks